src/app/core/components/left-nav-tree/left-nav-tree.component.ts

```diff
@@ -19,6 +19,10 @@ export class LeftNavTreeComponent implements OnChanges {
   }
 
   private mapItemToNodes(data: NavTreeData): TreeNode<NavTreeElement>[] {
+    const inL1 = data.elements.some(
+      e => !data.selectedElementId || e.hasChildren
+        && (e.route.id === data.selectedElementId || e.children?.find(c => c.route.id === data.selectedElementId))
+    );
```

Comment:
This check is a bit weird and no always valid I think. The cleanest way to know if we are on l1 is to add `onL1` boolean parameter to ` mapItemToNodes()` and to pass `true` when called on L18 and pass `false` on line 32.

Reply:
Fixed #1362